addon/store.js

```diff
@@ -194,4 +196,10 @@ var Store = Ember.Object.extend({
     }
 });
 
+if (!isModern) {
+    Store.reopenClass({
```

Comment:
I recalled an inject.service shim that respected this but now I can't find it so likely the project is obsolete.  Will remove this